### DISCUSSION TITLE
Disable/hide detail expand for specific rows

### DISCUSSION THREAD

**Discussion by amhassen97** [KevinVandy/material-react-table#212] [Q&A] [answered]
Hi

Is it possible to disable or hide the detail expand button/functionality for specific rows on the table?

Thanks for the help!

**Comment by KevinVandy** [comment, marked as answer]
Check out this example https://www.material-react-table.dev/?path=/story/features-detail-panel-examples--detail-panel-enabled-conditional

**Reply by amhassen97** [reply]
This is perfect, thanks for the quick response 👌👌!
